logbook-spring-webflux/src/main/java/org/zalando/logbook/spring/webflux/ServerResponse.java

```diff
@@ -34,7 +34,9 @@ public HttpHeaders getHeaders() {
 
     @Override
     public int getStatus() {
-        return Optional.ofNullable(response.getRawStatusCode()).orElse(200);
+        return Optional.ofNullable(response.getStatusCode())
```

Comment:
I think you missed one implementation in [org.zalando.logbook.spring.webflux.ClientResponse](https://github.com/zalando/logbook/blob/6b1168bbd922f02058344ab436c2d2230a0fd97d/logbook-spring-webflux/src/main/java/org/zalando/logbook/spring/webflux/ClientResponse.java#L27)

Reply:
Thanks, fixed. I added a `FIX-ME` comment to keep track of locations where we should discuss how to avoid NPE.